docs/source/csd.rst

```diff
@@ -0,0 +1,151 @@
+.. _csd-module:
+
+====================================
+Care Services Discovery Module (CSD)
+====================================
+
+.. contents::
+   :depth: 3
+
+The Care Services Discovery module allows MOTECH to consume information from a registry based on the `IHE Care Services Discovery Profile <http://wiki.ihe.net/index.php/Care_Services_Discovery>`_. The CSD registry contains information about health organizations, facilities, services and providers. MOTECH is able to consume the registry and represent it in MOTECH Data Services so the information is locally available for queries, updates and tasks.
+
+The CSD Standard and OpenHIE integration
+----------------------------------------
+The CSD is used by `OpenInfoMan <https://github.com/openhie/openinfoman>`_, which is the reference software for the `OpenHIE Health Worker Registry <https://ohie.org/health-worker-registry/>`_. This registry provides a big picture view of a country's health workforce as well as a single source for supporting patient/provider flows in a Health Information Exchange. More information on the CSD standard can be found in the `IHE Care Services Discovery Profile <http://wiki.ihe.net/index.php/Care_Services_Discovery>`_ and the OpenInfoMan implementation of that profile on their `GitHub page <https://github.com/openhie/openinfoman>`_.
+
+In some implementations, OpenInfoMan may be deployed within the OpenHIE stack behind a firewall. `OpenHIM <http://openhim.org/>`_ is the Interoperability layer for OpenHIE. It responds to CSD requests from point of service applications like MOTECH and is able to forward those requests to OpenInfoMan. The same requests to OpenInfoMan can be made to OpenHIM. OpenHIM being the system that provides access controls and auditing of requests.
+
```

Comment:
This last sentence "OpenHIM being the system that provides access controls and auditing of requests" is a fragment. You might just want to rephrase as "OpenHIM is the system that provides access controls and auditing of requests".